### PR TITLE
Allow the configuration of HOY_INVERTER_WATT

### PR DESCRIPTION
**Pull request by ms49434**
HOY_INVERTER_WATT should contain the power rating of the inverter.
HOY_MAX_WATT is used to limit the maximum output of the inverter and can be equal or lower than HOY_INVERTER_WATT.

Example 1:
A Hoymiles inverter HMS-2000-4T has a power rating of 2000 Wp.
The inverter shouldn't produce more than 600 W.

HOY_INVERTER_WATT = 2000
HOY_MAX_WATT = 600

Example 2:
A Hoymiles inverter HM-800-2T has a power rating of 800 Wp.
The maximum power should be the same as the power rating.

HOY_INVERTER_WATT = 800
HOY_MAX_WATT = 800

### CODE REVIEW

**Comment by reserve85** [comment]
Hello,

looks good, I would never have thought that this could be useful without battery operation, but why not.
Can you please increase the version in the config and script and add a short description in the changelog? Then I can push it into main.

Thank you for your support!

**Comment by ms49434** [comment]
Hello,

please double check and let me know if I missed something.

:)

**Review by reserve85 — CHANGES_REQUESTED** [review]
just add a fallback value so the new entry is just optional.

File: HoymilesZeroExport_Config.ini [hunk 1]
```diff
@@ -239,8 +239,10 @@ POWERMETER_MAX_POINT = 0
 # serial number of your inverter, if empty it is automatically read out of the API. If you have more than one inverter you should define the serial number here (prevents mix-up).
 SERIAL_NUMBER = 
 # power rating of your inverter
+HOY_INVERTER_WATT = 1500
```
[the remark this answers is not in the document]
Comment (ms49434) [reply HoymilesZeroExport_Config.ini]: Modifications have been applied.

**Review by reserve85 — APPROVED** [review]
thank you

**Pull request merged**